src/marine/rads_adt2ioda.py

```diff
@@ -41,37 +60,28 @@ def _read(self):
         lons = ncd.variables['lon'][:]
         lats = ncd.variables['lat'][:]
         vals = ncd.variables['adt_xgm2016'][:]
+        val_units = ncd.variables['adt_xgm2016'].units
+        Fill_val = ncd.variables['adt_xgm2016']._FillValue
+        scale_factor = ncd.variables['adt_xgm2016'].scale_factor
         units = ncd.variables['time_mjd'].units[-23:-4]
         reftime = dateutil.parser.parse(units)
         ncd.close()
 
-        valKey = vName, self.writer.OvalName()
-        errKey = vName, self.writer.OerrName()
-        qcKey = vName, self.writer.OqcName()
+        valKey = vName, iconv.OvalName()
+        errKey = vName, iconv.OerrName()
+        qcKey = vName, iconv.OqcName()
+        self.VarAttrs[ vName, iconv.OvalName()]['_FillValue'] = Fill_val
+        self.VarAttrs[ vName, iconv.OvalName()]['units'] = val_units
+        self.VarAttrs[ vName, iconv.OvalName()]['scale_factor'] = scale_factor
```

Comment:
Please add attributes "_FillValue" and "units" for the OerrName and OqcName versions of the variable.
* _FillValue for OerrName and OqcName variables should match what the data provider supplies (which are not necessarily the same as the OvalName variable, but typically are the same).
* Units for the OerrName variable can have the value from val_units (ie same as OvalName variable).
* Units for the OqcName variable should have the value 'unitless'.